Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/response/TrustMode.java

```diff
@@ -66,7 +66,13 @@ public TrustMode() {
     public PRPAIN201306UV02 processResponse(ResponseParams params) {
         LOG.debug("Begin TrustMode.processResponse()...");
         PRPAIN201306UV02 response = null;
-
+        int sonargithub = 1;
+        //TODO: remove this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://GFE-ONC-LDEV15.connect.org:9000/coding_rules#rule_key=squid%3AS1135)